integration/test-executions.js

```diff
@@ -12,7 +12,7 @@ var expect = chai.expect
   , st2client = require('../index')(config)
   ;
 
-var MINIMUM_ENTITIES = 3;
+var MINIMUM_ENTITIES = 1;
```

Comment:
The idea behind `MINIMUM_ENTITIES` was to ensure that endpoint actually returns a list of entities rather than a list of single entity. In retrospective, it looks a bit silly, but having it set to 1 negates the purpose of having it as a constant entirely.

There's three equally valid options here:
 - get it back to >1 and create missing entities in `before`
 - just hardcode the number and get rid of the constant
 - ignore silly me because this thing is not worth the mental effort at that point